packages/devtools_app/lib/src/framework/html_framework.dart

```diff
@@ -687,17 +690,17 @@ class HtmlToastAnimator {
 
   static const Duration extendedHideDelay = Duration(seconds: 10);
 
+  static const Duration infiniteHideDelay = Duration(days: 1);
+
   final CoreElement element;
 
-  void show({
-    Duration hideDelay = defaultHideDelay,
-    bool neverHide = false,
-  }) async {
+  void show({Duration hideDelay = defaultHideDelay}) async {
     await window.animationFrame;
 
     element.element.style.left = '0px';
 
-    if (neverHide) return;
+    // Skip creating the timer if the selected delay is [infiniteHideDelay].
```

Comment:
this check is super surprising unless you change infiniteHideDelay to really use double.infinity.

Reply:
used null as discussed offline